### PR TITLE
manila: Allow using keystone v3 API

### PR DESCRIPTION
**Pull request by vuntz**
cc @rhafer

### CODE REVIEW

File: chef/data_bags/crowbar/migrate/manila/011_add_max_header_line.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def upgrade(ta, td, a, d)
+  a["max_header_line"] = 16384
```
Comment (toabctl) [inline chef/data_bags/crowbar/migrate/manila/011_add_max_header_line.rb]: I think we usually use the default value so ```ta["max_header_line"]```

Comment (rhafer) [inline chef/data_bags/crowbar/migrate/manila/011_add_max_header_line.rb]: @vuntz Is this supposed to be backported to tex? Then we need the the ```unless a.has_key()``` thing here.

Comment (vuntz) [inline chef/data_bags/crowbar/migrate/manila/011_add_max_header_line.rb]: fixed

**Comment by rhafer** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**